assets/locales/en/translation.json

```diff
@@ -141,9 +141,8 @@
   "Get Timing": "Get Timing",
   "interactionsNotAvailable": "Interactions for this element may not be available",
   "usingXPathNotRecommended": "Using XPath locators is not recommended and can lead to fragile tests. Ask your development team to provide unique accessibility locators instead!",
-  "usingSwitchContextRecommended": "Webview context(s) detected; certain elements might only be accessible after switching to a webview context, which must be done in an Appium script. See http://appium.io/docs/en/writing-running-appium/web/hybrid/ for more information",
-  "usingWebviewContext": "Using the Webview inspector in Appium Inspector is less accurate in retrieving and selecting DOM elements in comparison to using the DevTools of Chrome and Safari.",
-  "contextSwitcher": "You can switch to a different context here. See http://appium.io/docs/en/writing-running-appium/web/hybrid/ for more information",
+  "noAdditionalContextsFound": "Appium has detected only the native context",
```

Comment:
Just wanted to put my 2 cents there: It would probably be too generic or not even correct to say "Appium has detected" as Appium is just an abstract server. Webview detection is done by the corresponding driver, for example UIA2 or XCUITest one.

Reply:
Fixed. I decided that instead of explicitly mentioning 'Appium driver' or something similar, it would still be fine to use even more generic wording 😄